src/python/sortingview/gui/extensions/mountainview/MVSortingComparisonView/ComparisonViewLauncher.tsx

```diff
@@ -0,0 +1,48 @@
+import { usePlugins } from "labbox-react"
```

Comment:
This comparison view launcher is implemented here, but it is empty for now, because we don't have any sorting comparison view plugins to launch.